cmd/keeper/keeper.go

```diff
@@ -1579,6 +1587,7 @@ func main() {
 
 func keeper(cmd *cobra.Command, args []string) {
 	var err error
+	validAuthMethods := map[string]bool{"md5": true, "password": true, "trust": true}
```

Comment:
I'm not sure if we should also add the password method since postgres doc suggests to use md5.